test/integration/framework/framework.go

```diff
@@ -79,3 +82,28 @@ func AfterSuite(testEnv *envtest.Environment) {
 	err := testEnv.Stop()
 	gomega.ExpectWithOffset(1, err).NotTo(gomega.HaveOccurred())
 }
+
+func DeleteCapacity(ctx context.Context, c client.Client, cap *kueue.Capacity) error {
+	if cap != nil {
+		if err := c.Delete(ctx, cap); err != nil && !apierrors.IsNotFound(err) {
+			return err
+		}
+	}
+	return nil
+}
+
+func DeleteNamespace(ctx context.Context, c client.Client, ns *corev1.Namespace) error {
```

Comment:
Document that this deletes all objects that we care about in the namespace.

Reply:
done